translations/static/js/theme.js

```diff
@@ -0,0 +1,27 @@
+(() => {
+  "use strict";
+  const prefersDarkMode = window.matchMedia(
+    "(prefers-color-scheme: dark)"
+  ).matches;
+  const defaultTheme = prefersDarkMode ? "dark" : "light";
+  const preferredTheme = localStorage.getItem("theme");
+  const toggleDarkMode = document.querySelector("#bd-theme");
+  const isDarkTheme = localStorage.getItem("theme") === "dark";
```

Comment:
Move this inside of the toggleDarkMode event listener, because in this implementation the toggle switch only works once.

Reply:
Then I will not be able to access those variables outside of the function

Reply:
Use the `preferredTheme` for `toggleDarkMode.checked` instead of `isDarkTheme` which seems to be the only instance of it used outside of the event listener.
```
toggleDarkMode.checked = preferredTheme === "dark" ? true : false;
```